test/util/oauthUtil.js

```diff
@@ -155,8 +163,8 @@ define(function(require) {
         if (opts.willFail) {
           throw err;
         } else {
-          // Should never hit this
-          expect(true).toBe(false);
+          expect('not to be hit').toBe(true);
+          console.log(err); // eslint-disable-line
```

Comment:
@rchild-okta re: previous comment. This is how I'm handling the errors for now. This is kinda weird logic anyway. It probably always makes sense to throw, so I'll change the fin